packages/direflow-component/src/services/materialUiStylesHandler.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { IDireflowPlugin } from '../types/DireflowConfig';
+
+const handleMaterialUiStyle = (app: React.FC<any> | React.ComponentClass<any, any>, plugins: IDireflowPlugin[] | undefined)
```

Comment:
This line exceeds the maximum allowed characters of 120.
Please break up the parameters to align vertically.
You can use the formatter for this - ESLint should give you some heads-up 🙂